morse_code.rb

```diff
@@ -0,0 +1,58 @@
+CHARS = {
+  '.-' => 'a',
+  '-...' => 'b',
+  '-.-.' => 'c',
+  '-..' => 'd',
+  '.' => 'e',
+  '..-.' => 'f',
+  '--.' => 'g',
+  '....' => 'h',
+  '..' => 'i',
+  '.---' => 'j',
+  '-.-' => 'k',
+  '.-..' => 'l',
+  '--' => 'm',
+  '-.' => 'n',
+  '---' => 'o',
+  '.--.' => 'p',
+  '--.-' => 'q',
+  '.-.' => 'r',
+  '...' => 's',
+  '-' => 't',
+  '..-' => 'u',
+  '...-' => 'v',
+  '.--' => 'w',
+  '-..-' => 'x',
+  '-.--' => 'y',
+  '--..' => 'z'
+}.freeze
+
+def decode_char(morse_char)
+  CHARS[morse_char].upcase
+end
+
+def decode_word(morse_word)
+  word = ''
+  word_array = morse_word
+  word_array.split.each { |i| word += decode_char(i) }
+  word
+end
+
+def decode_message(morse_message)
+  message_array = morse_message.split('   ')
+  message = ''
+  message_array.each { |i| message += "#{decode_word(i)} " }
+  message
```

Comment:
- [ ] **[OPTIONAL]** Your sentence decoder returns the string with whitespace. I would recommend you remove it.

  ![image](https://user-images.githubusercontent.com/81468675/222413433-e18b6cf8-d743-47cb-9ee6-985a5a7cbc78.png)